Project.cpp

```diff
@@ -6,8 +6,8 @@ class User{
     string Username;
     string Email;
     User(string username,string email){
-        Username = username;
-        Email = email;
+        this->Username = username;
```

Comment:
Consider removing `this->` when initializing class members in constructors to make the code cleaner and more readable. [medium]